apps/docs/src/app/documentation/core-helpers/sections-toolbar/sections-toolbar.component.html

```diff
@@ -30,7 +30,7 @@
                         </div>
                         <ul fd-nested-list>
                             <ng-container *ngFor="let item of section.content; trackBy: trackBySectionContent">
-                                <ng-container *ngIf="item.subItems; else rootListItem">
+                                <ng-container *ngIf="$any(item).subItems as itemSubItems; else rootListItem">
```

Comment:
Yeah, I considered this. Decided to go with `$any` because type is lost anyway since once it's passed as ngTemplateOutletContent 🤷‍♂️ And $any is actually being removed at compilation step, so it's slightly better

But if you ask, I cannot refuse you 🙂